tests/tests_threebody/test_cr3bp_char_quant.py

```diff
@@ -0,0 +1,27 @@
+"""
+@author: Dhruv Jain, Multi-Body Dynamics Research Group, Purdue University
+"""
+from astropy.tests.helper import assert_quantity_allclose
+
+from poliastro.threebody.cr3bp_char_quant import SystemChars
+
+
+def test_mu(p1, p2, expected_mu):
```

Comment:
Unfortunately these tests won't run, pytest expects test parameters to be fixtures or parametrized parameters. You can verify this yourself locally running

```
tox -e py39-fast
```

(replace `39` with `38` or `310` depending on your Python version)

or

```
tox -e py39 -- -k threebody
```

(to only run threebody tests)

If you want to run the same test for different values of a parameter, have a look at `@pytest.mark.parametrize`. We are using it in some other places in poliastro.

Reply:
I will focus on the cr3bp_char_quant.py for now and come back to the tests later.